libs/forecasting/src/lib/ForecastingDashboard/Controls/getForecastPrediction.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Cohort, JointDataset } from "@responsible-ai/core-ui";
+
+export async function getForecastPrediction(
```

Comment:
why do we need a special one for forecasting instead of using the regular prediction one?

Reply:
Because it's more efficient. We don't send data across, just the transformation description that applies to the whole time series.